test/environment.go

```diff
@@ -33,3 +34,17 @@ func (e *TestEnvironment) Now() time.Time {
 func (e *TestEnvironment) SetNow(now time.Time) {
 	e.now = now
 }
+
+func (e *TestEnvironment) FindLocations(string, utils.LocationLevel, *utils.Location) ([]*utils.Location, error) {
```

Comment:
so that this env type can be used as a RunEnvionment which is required by FieldValues.setValue because it potentially looks up locations.